index.html

```diff
@@ -1,277 +1,558 @@
+<h2 id="features">Features</h2>
+<ul>
+<li>Toggle between HTML, plain text views as well as view email headers</li>
+<li>Test responsive emails with resizable preview pane available for various screen sizes</li>
+<li>Ability to receive and view email attachments</li>
+<li>WebSocket integration keeps the interface in sync once emails are received</li>
+<li>Command line interface for configuring SMTP and web interface ports</li>
+<li>Ability to relay email to an upstream SMTP server</li>
+</ul>
+<h2 id="ideas">Ideas</h2>
+<p>If you&#39;re using MailDev and you have a great idea, I&#39;d love to hear it. If you&#39;re not using MailDev because it lacks a feature, I&#39;d love to hear that too. Add an issue to the repo <a href="https://github.com/maildev/maildev/issues/new">here</a>.</p>
+<h2 id="contributing">Contributing</h2>
+<p>Any help on MailDev would be awesome. There is plenty of room for improvement. Feel free to <a href="https://github.com/maildev/maildev/issues/new">create a Pull Request</a> from small to big changes.</p>
+<p>To run <strong>MailDev</strong> during development:</p>
+<pre><code>npm install
+npm run dev
+</code></pre>
+<p>The &quot;dev&quot; task will run MailDev using nodemon and restart automatically when
+changes are detected. On <code>*.scss</code> file save, the css will also be recompiled.
+Using <code>test/send.js</code>, a few test emails will be sent every time the application
+restarts.</p>
+<p>The project uses the <a href="https://standardjs.com">JavaScript Standard coding style</a>.
+To lint your code before submitting your PR, run <code>npm run lint</code>.</p>
+<p>To run the test suite:</p>
+<pre><code>$ npm test
+</code></pre>
+<h2 id="changelog"><a href="https://github.com/maildev/maildev/releases">Changelog</a></h2>
+<h2 id="thanks">Thanks</h2>
+<p><strong>MailDev</strong> is built on using great open source projects including
+<a href="http://expressjs.com">Express</a>,
+<a href="http://angularjs.org/">AngularJS</a>,
+<a href="http://fontawesome.io/">Font Awesome</a> and two great projects from
+<a href="https://github.com/andris9">Andris Reinman</a>:
+<a href="https://github.com/nodemailer/smtp-server">smtp-server</a>
+and <a href="https://github.com/nodemailer/mailparser">mailparser</a>.
+Many thanks to Andris as his projects are the backbone of this app and to
+<a href="http://mailcatcher.me/">MailCatcher</a> for the inspiration.</p>
+<p>Additionally, thanks to all the awesome <a href="https://github.com/maildev/maildev/graphs/contributors">contributors</a>
+to the project.</p>
+<h2 id="license">License</h2>
+<p>MIT</p>
+<!-- README content end -->
+
+    </article>
+  </div>
+
+
+  <footer>
+
+  </footer>
+
+
+
+  <footer class="bg-brand py-12  text-center">
+    <div class="container max-w-screen-md mx-auto px-4">
+      <p class="text-white font-medium">This project is maintained by <a class="underline font-semibold" href="http://danfarrelly.nyc">Dan Farrelly</a></p>
```

Comment:
```suggestion
```

@DominikSerafin, we can remove this line too. With it being moved to the `maildev` github org and folks like you making the major contributions, this feels outdated 😁